drivers/input/Kconfig.kbd_matrix

```
@@ -20,6 +20,21 @@ config INPUT_KBD_MATRIX_16_BIT_ROW
 	  Use a 16 bit type for the internal structure, allow using a matrix
 	  with up to 16 rows if the driver supports it.
 
+config INPUT_KBD_MATRIX_STATE
```

Non-blocking suggestion
```suggestion
config INPUT_SHELL_KBD_MATRIX_STATE
```